### ISSUE TITLE
Train CNN model can't convergence

### ISSUE THREAD

**Issue by wangxingxing** [apache/incubator-mxnet#2524]
I try use train_imagenet.py  train cnn model on imagenet dataset, but the model do't convergence, the train log:

> 2016-06-24 19:48:47,174 Node[0] Start training with [gpu(0)]
> 2016-06-24 19:49:17,983 Node[0] Epoch[0] Batch [10]     Speed: 868.98 samples/sec       Train-accuracy=0.001953
> 2016-06-24 19:49:17,984 Node[0] Epoch[0] Batch [10]     Speed: 868.98 samples/sec       Train-cross-entropy=10.274762
> 2016-06-24 19:49:21,588 Node[0] Epoch[0] Batch [20]     Speed: 710.26 samples/sec       Train-accuracy=0.000000
> 2016-06-24 19:49:21,588 Node[0] Epoch[0] Batch [20]     Speed: 710.26 samples/sec       Train-cross-entropy=6.910368
> 2016-06-24 19:49:25,179 Node[0] Epoch[0] Batch [30]     Speed: 713.03 samples/sec       Train-accuracy=0.001953
> 2016-06-24 19:49:25,179 Node[0] Epoch[0] Batch [30]     Speed: 713.03 samples/sec       Train-cross-entropy=6.907778
> 2016-06-24 19:49:28,785 Node[0] Epoch[0] Batch [40]     Speed: 710.12 samples/sec       Train-accuracy=0.000000
> 2016-06-24 19:49:28,786 Node[0] Epoch[0] Batch [40]     Speed: 710.12 samples/sec       Train-cross-entropy=6.907757
> 2016-06-24 19:49:32,353 Node[0] Epoch[0] Batch [50]     Speed: 717.71 samples/sec       Train-accuracy=0.001172
> 2016-06-24 19:49:32,353 Node[0] Epoch[0] Batch [50]     Speed: 717.71 samples/sec       Train-cross-entropy=6.907731
> 2016-06-24 19:49:35,948 Node[0] Epoch[0] Batch [60]     Speed: 712.21 samples/sec       Train-accuracy=0.001563
> 2016-06-24 19:49:35,948 Node[0] Epoch[0] Batch [60]     Speed: 712.21 samples/sec       Train-cross-entropy=6.907753
> 2016-06-24 19:49:39,524 Node[0] Epoch[0] Batch [70]     Speed: 715.80 samples/sec       Train-accuracy=0.001953
> 2016-06-24 19:49:39,525 Node[0] Epoch[0] Batch [70]     Speed: 715.80 samples/sec       Train-cross-entropy=6.907798

i use this code to generate train.lst

> train_list=importdata('/home/wangxx/code/caffe/data/ilsvrc12/train.txt');
> fid0 = fopen(fullfile(root_path, 'ilsvrc2012_train_caffe_shuffle.txt'), 'w');
> fid1 = fopen(fullfile(root_path, 'ilsvrc2012_train_mxnet_shuffle.txt'), 'w');
> randindex = randperm(length(train_list.data));
> for i = 1 : length(train_list.data)
>     fprintf(fid0, '%s %d\n', cell2mat(train_list.textdata(randindex(i))), train_list.data(randindex(i)));
>     fprintf(fid1, '%d\t%d\t%s\n', randindex(i) - 1, train_list.data(randindex(i)), cell2mat(train_list.textdata(randindex(i))));
> end
> fclose(fid0);
> fclose(fid1);

and use this train list file to generate LMDB and REC dataset file.
the caffe train loss is convergence, and get 35% top5 accuracy after 1 epoch use alexnet.

The hyper-parameter train caffe and mxnet is same, and i had train difference model like: alexnet, vgg, inception-v1. 

and i had try use im2rec.py to generate train.lst and REC file, still not convergence.
@tqchen

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
